src/SignalR/clients/csharp/Client.SourceGenerator/src/gen/Microsoft.AspNetCore.SignalR.Client.SourceGenerator.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Source Generator for ASP.NET Core SignalR Client</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <SuppressDependenciesWhenPacking>false</SuppressDependenciesWhenPacking>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis.CSharp" PrivateAssets="all" />
```

Comment:
If Microsoft.CodeAnalysis.CSharp is okay with dropping internal attributes in assemblies with a source generator, why can't we?